examples/eve/agent/channels/eve.ts

```
@@ -0,0 +1,15 @@
+import { eveChannel } from "eve/channels/eve";
```

ESLint

[error] Extra semicolon.

(semi)